### ISSUE TITLE
README needs more information on input data requirements in Getting Started Section 2

### ISSUE THREAD

**Issue by libertyh** [alexrockhill/pd-parser#8]
In the README.rst **Getting Started** section, step 2 is to try out the function on your own data. I think there needs to be more information about what that data should look like either in this README or in the docs, as it's not currently clear which file types are supported and how exactly the channels should be assigned before using the tool. The user could run the examples in step 1, but I think it's nice to have some of this information already available so users can determined whether the tool will be useful to them.  For me, I like to have a specific recipe of "Here are the exact files you'll need, here are some example files for you to look at and compare to your own, and this is the order of the function calls you should use to get started". I think the last part is done well by your notebook example, so perhaps it would be redundant to include in the README. Still, I think having the formats and some example files will help your users to dive right into using the code.

For example, `parse_pd.py` has a function `_read_raw()` that can take in .fif, .edf, .bdf, .vhdr, or .set files. This information should be stated in the README/docs in **Getting Started**, as well as something to note that the photodiode channel name should be contained in that file and will be used as input to `find_pd_params()`. 

Although the behavior .tsv file is made by running the examples in section 1 of **Getting Started**, I think it would be helpful to keep that file in an examples directory so the user can reference it without having to run the notebook and find the tsv file in a tmp directory. 

So, to summarize, my thoughts are to provide a short list in the README, as follows:

**Files you need**:

* Neural data in .fif, .edf, .bdf, .vhdr, or .set format. (any other expectations about directory structure, etc) (example link)
* behavior.tsv (example link)

**How to run**:

* See example notebooks for extracting parameters for pd_parser(), using these to detect events, and saving to BIDS compatible format

**Comment by alexrockhill** [comment]
That would be good to add, I'll go ahead and change that

**Comment by alexrockhill** [comment]
I would note that the photodiode channel (in reference to `find_pd_params` and other functions) is an optional argument and if you don't know it when you call the function, the data will be plotted and user input will be requested.

**Comment by alexrockhill** [comment]
I incorporated your suggestions in d98990fe0291bfbaa015aee3852c79b907eaa490, thanks!

**Comment by libertyh** [comment]
Awesome, thanks! I think this will make things a lot easier.
